cmd/clusterctl/clusterdeployer/bootstrap/kind/kind.go

```diff
@@ -32,7 +32,7 @@ const (
 
 var (
 	// ignoredOptions lists the options not supported by delete and kubeconfig-path.
-	ignoredOptions = []string{"config", "image", "retain"}
+	ignoredOptions = []string{"config", "image", "retain" }
```

Comment:
There is still a diff here which would cause a merge conflict with your other PR

Reply:
Fixed :-)